R/tm_g_distribution.R

```diff
@@ -1414,9 +1415,9 @@ srv_distribution <- function(id,
         card$append_text("Statistics table", "header3")
         card$append_table(decorated_output_summary_q()[["summary_table"]])
         tests_error <- tryCatch(expr = tests_r(), error = function(e) "error")
-        if (inherits(tests_error, "data.frame")) {
+        if (!identical(tests_error, "error")) {
```

Comment:
To review: Test table is always added to the report. I believe this was the initial goal of this module